src/components/Autocomplete/AutocompleteBase/index.tsx

```diff
@@ -0,0 +1,24 @@
+import {
+    Autocomplete as MaterialAutocomplete,
+    AutocompleteProps as MaterialAutocompleteProps,
+} from "@material-ui/lab";
+
+export interface AutocompleteBaseProps<
+    T,
+    Multiple extends boolean | undefined,
+    DisableClearable extends boolean | undefined,
+    FreeSolo extends boolean | undefined
+> extends MaterialAutocompleteProps<T, Multiple, DisableClearable, FreeSolo> {}
+
+const AutoCompleteBase = <
+    T extends {},
+    Multiple extends boolean | undefined,
+    DisableClearable extends boolean | undefined,
+    FreeSolo extends boolean | undefined
+>(
```

Comment:
```suggestion
const AutoCompleteBase = <AutocompleteBaseProps>(
```

Reply:
I'm not sure about this one but please try to not have duplicate code in one file